.github/workflows/main.yml

```diff
@@ -39,13 +39,17 @@ jobs:
           submodules: recursive
 
       - name: Go build
-        run: CGO_ENABLED=0 GOOS=linux go build -a -o artifacts/polygon-sdk .
+        run: CGO_ENABLED=1 GOOS=linux go build -race -a -o artifacts/polygon-sdk .
```

Comment:
I thought the point of having a separate build routine `Go build with race` is that the regular `Go build` would be _without_ the _race_ flag?

Reply:
Fixed it, Commit [link](https://github.com/0xPolygon/polygon-sdk/pull/207/commits/4ae18f2be64714f933d5e5b421737c831b2e9e6a)